src/main/scala/amba/axi4/Fragmenter.scala

```diff
@@ -62,7 +61,7 @@ class AXI4Fragmenter()(implicit p: Parameters) extends LazyModule
         val len  = Mux(busy, r_len,  a.bits.len)
         val addr = Mux(busy, r_addr, a.bits.addr)
 
-        val lo = if (lgBytes == 0) UInt(0) else addr(lgBytes-1, 0)
+        if (lgBytes == 0) UInt(0) else addr(lgBytes-1, 0)
```

Comment:
Should this line just be deleted entirely? is it actually not used?

Reply:
Same here. Scalafix removes "val lo" due to non-existant referrals in the local scope. However, removing the whole line is prohibited, since `addr(lgBytes-1,0)` may incur some side-effects (like db query, http, console, etc) and thus can't be eliminated. 

That's the idea of refactoring. Remove all unnecessary calls/vals/imports to simplify the code base and save on JVM allocs in case of val